### PR TITLE
chore(ClickHouse): add celery task to monitor CH schema drift

### PR DESCRIPTION
**Pull request by guidoiaquinti**
## Problem
This PR adds a Celery task (running every minute) to monitor for ClickHouse schema drift in a multi replica environments.

The task currently logs & send a metric to `StatsD`.

## Changes
N/A

## How did you test this code?
I've tested it locally + added unit tests (see PR)

### CODE REVIEW

File: posthog/tasks/check_clickhouse_schema_drift.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    # Parse query rows and put them in a Dict like:
+    # {
+    #   "table1": {"schema1": ["host1", "host2", "host3"]},
+    #   "table2": {
+    #     "schema2": ["host1"],
+    #     "schema2-different": ["host2"],
+    #     "schema2-different-bis": ["host3"]},
+    # }
+    tables = {}  # type: Dict
+    for item in clickhouse_schema:
```
Comment (hazzadous) [inline posthog/tasks/check_clickhouse_schema_drift.py:72]:
```suggestion
    for table_name, schema, node_name in clickhouse_schema:
```

File: posthog/tasks/check_clickhouse_schema_drift.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        clickhouse_schema = get_clickhouse_schema()
+    except ClickhouseError:
+        logger.error("check_clickhouse_schema_drift_error", exc_info=True)
+        return  # no need to raise as we execute this often
+
+    drift = get_clickhouse_schema_drift(clickhouse_nodes, clickhouse_schema)
+
+    logger.info("check_clickhouse_schema_drift", table_count=len(drift), tables=drift)
+
+    # Send to stasd the total count of drifting tables as well as a metric for each table
```
Comment (hazzadous) [inline posthog/tasks/check_clickhouse_schema_drift.py:107]:
```suggestion
    # Send to statsd the total count of drifting tables as well as a metric for each table
```

**Review by hazzadous — COMMENTED** [review]
Looks good.

Test on check_clickhouse_schema_drift would be good also.

File: posthog/celery.py [hunk 3]
```diff
@@ -301,6 +304,13 @@ def status_report():
     status_report()
 
 
+@app.task(ignore_result=True)
+def monitoring_check_clickhouse_schema_drift():
+    from posthog.tasks.check_clickhouse_schema_drift import check_clickhouse_schema_drift
```
[the remark this answers is not in the document]
Comment (guidoiaquinti) [reply posthog/celery.py:309]: It should, I've just used the same pattern I found for the other tasks.

**Review by hazzadous — APPROVED** [review]

**Pull request merged**